### ISSUE TITLE
[Bug]: Parallel Test Error

### ISSUE THREAD

**Issue by huangdijia** [pestphp/pest#1085]
### What Happened

```shell
In WorkerCrashedException.php line 41:
                                                                                                                                                                                                           
  The test "PARATEST='1' TEST_TOKEN='2' UNIQUE_TEST_TOKEN='2_65bcb9f885603' tests/Support/FunctionsTest.php" failed.                                                                                       
                                                                                                                                                                                                           
  Exit Code: 1(General error)                                                                                                                                                                              
                                                                                                                                                                                                           
  Working directory: /Users/hdj/github/friendsofhyperf/components                                                                                                                                          
                                                                                                                                                                                                           
  Output:                                                                                                                                                                                                  
  ================                                                                                                                                                                                         
                                                                                                                                                                                                           
     ERROR  Fatal error: Uncaught ArgumentCountError: Too few arguments to function ParaTest\WrapperRunner\ApplicationForWrapperWorker::__construct(), 7 passed in /Users/hdj/    
  github/friendsofhyperf/components/vendor/pestphp/pest/bin/worker.php on line 76 and exactly 8 expected in /Users/hdj/github/friendsofhyperf/components/vendor/brianium/paratest/src/WrapperRunner/Appli  
  cationForWrapperWorker.php:58        

```

### How to Reproduce

BC in paratestphp/paratest 7.4.0

https://github.com/paratestphp/paratest/compare/v7.3.1...v7.4.0#diff-8c135a74283c42cb5124bcee651b1f309e0c00484180779d1c49cca46405256dR66

### Sample Repository

_No response_

### Pest Version

v2.33.2

### PHP Version

8.2.15

### Operation System

macOS

### Notes

_No response_

**Comment by devajmeireles** [comment]
It seems to only happen with PHP above 8.1. My 8.1 pipeline passes, but 8.2 and 8.3 don't: https://github.com/tallstackui/tallstackui/actions/runs/7757533863

**Comment by nunomaduro** [comment]
Fixed: v2.33.3.
